frontend/.env.local.example

```
@@ -15,6 +15,8 @@ NEXT_PUBLIC_SANITY_API_VERSION=2026-03-23
 NEXT_PUBLIC_SANITY_PROJECT_ID=your-project-id
 NEXT_PUBLIC_SANITY_DATASET=development
 SANITY_API_READ_TOKEN=your-token
+# Required in every environment. Signs OG image URLs so callers cannot create arbitrary render work.
+OG_IMAGE_SECRET=generate-a-long-random-value
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OG_IMAGE_SECRET key should go before the SANITY_API_READ_TOKEN key

(UnorderedKey)